content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,43 @@ product form.
 
 .. _sales/advanced-price-rules:
 
+Discounts
+=========
+
+The :guilabel:`Discounts` feature allows the ability to set a discount or increase the price
+on *individual items* in a sales order. This is calculated as a percentage of that product's sales
+price.
+
+To access discounts, navigate to the *Sales* app setting page
+(:menuselection:`Sales app --> Configuration --> Settings`), and click the :guilabel:`Discounts`
+checkbox, then click :guilabel:`Save`.
+
+.. image:: pricing/discounts-option.png
+   :align: center
+   :alt: Click the checkbox labeled `Discounts` to activate Discounts.
+
+After the setting has been activated, navigate to the desired quotation by
+going to :menuselection:`Sales app --> Orders --> Quotations` at the top of the page. Once there, click on the
+desired quote from the list.
```

Comment:
Line over 100 chars, just need to break lines to keep a consistent line length. This just makes reviewing docs easier when line length is consistent :)
```suggestion
After the setting has been activated, navigate to the desired quotation by going to
:menuselection:`Sales app --> Orders --> Quotations` at the top of the page. Once there, click on
the desired quote from the list.
```